gufe/tokenization.py

```diff
@@ -126,6 +128,59 @@ def process(self, msg, kwargs):
         return msg, kwargs
 
 
+def new_key_added(dct, new_key, default):
```

Comment:
all of these functions need a docstring to both document what they do and when they're used